src/acl/external/LDAP_group/ext_ldap_group_acl.cc

```diff
@@ -188,7 +188,7 @@ static void
 squid_ldap_set_referrals(LDAP * ld, int referrals)
 {
     if (referrals)
-        ld->ld_options |= ~LDAP_OPT_REFERRALS;
+        ld->ld_options |= LDAP_OPT_REFERRALS;
```

Comment:
AFAICT, I saw this change in #2167. Should it be removed from that PR? That PR concern applies to this and similar changes in this PR: How does this fix affect existing ext_ldap_group_acl users? If the helper is going to produce different results, we should warn admins. Otherwise, PR description should explicitly state that the fix does not change associated helper behavior.

Reply:
Will deal with these in invidiaul PRs